cmake/Hunter/core_packages.cmake

```diff
@@ -19,3 +19,5 @@ hunter_add_package(intx)
 hunter_add_package(Microsoft.GSL)
 hunter_add_package(nlohmann_json)
 hunter_add_package(ethash)
+hunter_add_package(gRPC)
+hunter_add_package(Protobuf)
```

Comment:
These 2 should go into `extra_packages`, not `core_packages`. Silkworm core is compatible with WebAssembly, while gRPC and Protobuf are probably not.

Reply:
Yeah right.